Framework/NexusGeometry/src/NexusGeometrySave.cpp

```diff
@@ -682,8 +682,6 @@ class NexusGeometrySaveImpl {
 
 public:
   explicit NexusGeometrySaveImpl(Mode mode) : m_mode(mode) {}
```

Comment:
Currently this class definition goes public, private public. Could we combine it all into a single private and public section (as it reads like two separate classes)

Additionally is there any reason we now support copying? 
Looking at the existing tests we currently don't use copies (might have missed something), and I would say that if there were copies created it's probably because someone didn't pass by ref

Reply:
also caused by a previous merge, fixed.